.github/workflows/deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+        with:
+          fetch-depth: 0
+
+      - name: Install and Build 🔧
+        run: |
+          npm install
+          npm run build
+          cd demo
+          npm install
+          npm run build
+
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@4.1.3
+        with:
+          branch: main
+          folder: demo/dist
```

Comment:
```suggestion
          folder: demo/dist
          target-folder: main
```

Reply:
Added this:
https://github.com/votemike/react-accordion/pull/10

Reply:
What benefit does this have?

Reply:
If we don't put this in it's own folder, then when deploying, it will delete all the PR folders

Reply:
I think I'd prefer to just not clean the folder than have a redirect.
What do you think?

Reply:
I get that, but what if the following scenario happens:

1. You commit a new file `test.png` (which get's deployed to `votemike.github.io/react-accordion/test.png`)
2. You commit a delete file `test.png` (which is not removed from github pages)
3. Your code still looks for a `test.png` file which works on github pages
4. Someone clones the repo, but the `test.png` file isn't there

Either way I think it needs to be either:
1. In it's own folder
2. or clean: false

Reply:
Actually, wait....

What about this `clean-exclude` thing?

<img width="670" alt="Screenshot 2022-10-05 at 12 06 41" src="https://user-images.githubusercontent.com/92019586/194046561-cb5dd151-bda8-4bf6-9915-be78b2f70fd9.png">

Could we create a pattern to match the git SHA sums?

Reply:
`\b[0-9a-f]{5,40}\b`?